main.py

```diff
@@ -13,9 +13,17 @@
 
 x = windowWidth / 2
 y = windowHeight / 2
+
+# initialize player
+player = pygame.Rect(x, y, 40, 40)
+
+# intitialize coin
+coin = pygame.Rect(100, 100, 16, 16)
```

Comment:
You initialize a `coin` object as a rectangle here, but never use it later. I suggest to drop this.
The actual coin that is drawn later is a circle and not a rectangle anyway.